CHANGELOG.md

```diff
@@ -40,6 +40,14 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 - ROX-22701: Prevent deleting default policies through the API
 
 - ROX-20723: Remove monorepo substructure under `ui/` directory and switch from yarn v1 to npm for package management. Use `npm run` in place of `yarn` commands.
+- ROX-25981: Scanner V4 now fetches vulnerability data from [Red Hat's VEX files](https://security.access.redhat.com/data/csaf/v2/vex/) instead of [Red Hat's OVAL feed](https://security.access.redhat.com/data/oval/v2/) for RPMs installed in RHEL-based image containers.
+  - Fixed vulnerabilities affecting RHEL-based images are no longer identified by the respective RHSA, RHBA, nor RHEA anymore. Instead, they will be identified by CVE.
+    - This will also apply to vulnerabilities obtained from the [CVE map](https://security.access.redhat.com/data/metrics/cvemap.xml) (used for container-first scanning).
+    - This may potentially disrupt policies created around RHSAs, for example, but for most users, this change is much preferred over the previous results.
+  - Scanner V4 now only considers vulnerabilities affecting Red Hat products dated back to 2014.
+    - Previously when reading Red Hat's OVAL data, the vulnerabilities dated back to pre-2000; however, most of those vulnerabilities were irrelevant.
+      We can found most, if not all, vulnerabilities prior to 2014 are not relevant these days, so they are not considered.
```

Comment:
No empirical evidence, but just looking at scan results over the years we don't tend to see anything over a few years old. I know 2014 was chosen because it was when [Heartbleed](https://en.wikipedia.org/wiki/Heartbleed) came out.

https://www.cvedetails.com/ has a chart `Vulnerabilities by type & year` which goes back to 2014, and you can see the number of vulnerabilities skyrockets between 2016 and 2017 and then again between 2021 and 2022. They seem to only go back 10 years, but perhaps there is a reason for that. Lastly, this is hardcoded into ClairCore, so we cannot control it from here. I could remove the part of the comment that says "why" and just state the fact that it only goes back to 2014 without an explanation so we aren't on the hook for backing that statement